# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderTextureHostWrapper.h
# Commit: 49ae9de85ab9
# Full Hash: 49ae9de85ab90bf13b9c6bc2c8b6bb540140c90d
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-06 04:14:34
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Support using D3D11 textures as external compositor surfaces with D3D11 compositor. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95800
# ==============================================================================

diff -r 91d8f0f829fd -r 49ae9de85ab9 gfx/webrender_bindings/RenderTextureHostWrapper.h
--- a/gfx/webrender_bindings/RenderTextureHostWrapper.h	Thu Nov 05 20:18:13 2020 +0000
+++ b/gfx/webrender_bindings/RenderTextureHostWrapper.h	Thu Nov 05 19:27:23 2020 +0000
@@ -35,6 +35,7 @@
   void ClearCachedResources() override;
   RenderMacIOSurfaceTextureHost* AsRenderMacIOSurfaceTextureHost() override;
   RenderDXGITextureHost* AsRenderDXGITextureHost() override;
+  RenderDXGIYCbCrTextureHost* AsRenderDXGIYCbCrTextureHost() override;
 
   // RenderTextureHostSWGL
   size_t GetPlaneCount() const override;
